src/main/target/BEEROTORF4/target.c

```diff
@@ -27,14 +27,14 @@
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     DEF_TIM(TIM9, CH2,  PA3, TIM_USE_PPM,                       0, 0), // PPM IN
 
-    DEF_TIM(TIM1, CH2N, PB0, TIM_USE_MOTOR,                     0, 0), // M1 - DMA2_ST6
-    DEF_TIM(TIM8, CH3N, PB1, TIM_USE_MOTOR,                     0, 1), // M2 - DMA2_ST4
-    DEF_TIM(TIM2, CH2,  PA1, TIM_USE_MOTOR,                     0, 0), // M3 - DMA1_ST6
-    DEF_TIM(TIM5, CH1,  PA0, TIM_USE_MOTOR,                     0, 0), // M4 - DMA1_ST2
-    DEF_TIM(TIM3, CH1,  PC6, TIM_USE_MOTOR,                     0, 0), // M5 - DMA1_ST4 (conflicts with SDCard, switch off SDCard DMA if used for DShot)
-    DEF_TIM(TIM8, CH2,  PC7, TIM_USE_MOTOR,                     0, 0), // M6 - DMA2_ST3 (doesn't work for DShot)
-    DEF_TIM(TIM3, CH2,  PB5, TIM_USE_MOTOR,                     0, 0), // M7 - DMA1_ST5 (doesn't work for DShot)
-    DEF_TIM(TIM4, CH4,  PB9, TIM_USE_MOTOR,                     0, 0), // M8 (no DMA, doesn't work for DShot)
+    DEF_TIM(TIM8, CH2N, PB0, TIM_USE_MOTOR,                     0, 0), // M1 - DMAR2_ST1 (DMA2_ST2)
```

Comment:
What does `DMAR2_ST1` mean? I couldn't find such entity in STM reference manual.

Reply:
Seems like it's just `DMA2_ST1`, as it corresponds to `TIM8_UP`.

Reply:
That, and me being too lazy to type `DMAR: DMA`